File: Java-Core/src/main/java/presto/error/PrestoError.java
package presto.error;

public class PrestoError extends Exception {

	private static final long serialVersionUID = 1L;

	protected PrestoError() {
	}
	
	protected PrestoError(String message) {
		super(message);
	}

	protected PrestoError(Exception e) {
		super(e);
	}


	
}

File: Java-CodeServer/src/main/resources/js/prompto/error/PrestoError.js
function PrestoError() {
    var tmp = Error.apply(this, arguments);
    this.message = tmp.message
    tmp.name = this.name = 'PrestoError'
    Object.defineProperty(this, "stack", {
        get: function () {
            return tmp.stack
        }
    });
    return this;
}


exports.PrestoError = PrestoError;